src/pybind/mgr/pg_autoscaler/module.py

```diff
@@ -301,7 +301,7 @@ def _get_pool_status(
 
             adjust = False
             if (final_pg_target > p['pg_num_target'] * threshold or \
-                final_pg_target <= p['pg_num_target'] / threshold) and \
```

Comment:
changed the part because feel like it is not right to trigger adjusting Pgs in the case final_pg_target == p['pg_num_target'] since imagine if threshold is 1.0, then they would be equal and won't need adjusting.